src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java

```diff
@@ -16,6 +16,7 @@
  
```

Comment:
General requirement for each Java class:
- Add class header
- Add Javadoc comment